contracts/Auction.sol

```diff
@@ -96,16 +97,42 @@ contract Auction {
     }
 
 
-    function sendFunds(address x, address y, fixed128x10 cost) public {
+    function sendFunds(address x, address y) public {
 
     }
 
     /*
         Place a bid on the item. The bid is added to the history of bids list. Incoming bid must be higher than the highest bid otherwise reject the bid. If incoming bid
         is higher than highest bid, then the highest bidder's funds should be released back to he/she and update highest bid.
     */
-    function bid(address bidder, uint bid_amount) payable public {
-        bid_history.push(bid_record(bidder, bid_amount, block.timestamp));
+    function bid() payable public {
+        require(
+            block.timestamp <= end_time,
```

Comment:
shall we also add the condition check for `ended` variable

Reply:
uh maybe not sure yet. we might not even need the ended variable since we have the end_time

Reply:
what if the owner close the auction, do we need another variable to handle this case ?